src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -223,6 +223,10 @@ export default {
       const uniqueMatches = [...new Map(matches.map(match => [match.path, match])).values()];
       return orderSymbolsByPriority(uniqueMatches).slice(0, MAX_RESULTS);
     },
+    placeholderText() {
```

Comment:
Maybe we could make this a prop where the default value is the original placeholder text since we don't yet have a use case for using this modal without a technology context? By making it a prop, we can still make it be flexible to other apps using this component where that isn't strictly true.